meta-resin-common/recipes-containers/docker-disk/files/entry.sh

```diff
@@ -10,7 +10,7 @@ PARTITION_SIZE=${PARTITION_SIZE:=1024}
 
 # Create a BTRFS disk image
 dd if=/dev/zero of=/export/data_disk.img bs=1M count=$PARTITION_SIZE
-mkfs.btrfs /export/data_disk.img
+mkfs.btrfs --mixed --metadata=single /export/data_disk.img
```

Comment:
FWIW I've seen a device out of space with the following:
```
Data,single: Size:1.80GiB, Used:1.74GiB
   /dev/mmcblk2p6          1.80GiB

Metadata,DUP: Size:658.00MiB, Used:166.23MiB
   /dev/mmcblk2p6          1.29GiB
```
ie 1.8GiB take to store 1.74GiB of data, and 1.29GiB taken to store 166.23MiB of metadata, needless to say I'm all for unlocking some of that 1.1GiB (~35%!) that is being blocked from usage